Makefile

```diff
@@ -55,8 +55,7 @@ help: ## Display this help.
 
 .PHONY: build
 build: .generate.timestamp vet lint ## Build the KubeSAN image.
-	go mod download
-	go build -o bin/kubesan cmd/main.go
+	go build -mod=vendor -o bin/kubesan cmd/main.go
```

Comment:
How does one update the vendor/ directory (such as when intentionally deciding to upgrade to a newer version of a dependency)? Is there a new make target for that?  Meanwhile, the vendor/ directory makes the overall git repo a lot larger, even if it is now safer in that it no longer has to rely on external repositories (or worse supply-chain MitM attackers), so I'm okay with it.  But vendoring should be its own change, independent from the rest of this commit.

Reply:
When I split this out I'll add a make target for it but tl;dr is `go mod tidy && go mod vendor` will do your updates for you